population/create_glpi_computer_redfish.py

```diff
@@ -594,15 +596,23 @@ def post_to_glpi(  # noqa: C901
             glpi_fields_list.append(glpi_fields)
             api_range += api_increment
 
+    comment = None
+    COMPUTER_ID = None
     for glpi_fields in glpi_fields_list:
         for glpi_computer in glpi_fields.json():
             if glpi_computer["uuid"] == uuid:
                 global PUT
-                global COMPUTER_ID
                 PUT = True
                 COMPUTER_ID = glpi_computer["id"]
+                comment = glpi_computer["comment"]
                 break
 
+    # Add BMC Address to the Computer
+    plugin_response = check_fields(session, urls.BMC_URL)
```

Comment:
If the plugin isn't installed, the response will be `['ERROR_RESOURCE_NOT_FOUND_NOR_COMMONDBTM', 'resource not found or not an instance of CommonDBTM; view documentation in your browser at <URL>/#ERROR_RESOURCE_NOT_FOUND_NOR_COMMONDBTM']`

This behavior is accounted for in [`update_bmc_address()`](https://github.com/redhat-eets/glpi-helper-scripts/pull/23/files#diff-140fec13b178c343d8b749cd14258d6909fdcf42ffb93e22e7d9ffbe3a76b7e0R822) by adding the address to the comments field rather than the plugin field.